tests/build.go

```diff
@@ -187,6 +190,42 @@ RUN busybox | head -1 | grep v1.35.0
 	require.NoError(t, cmd.Run())
 }
 
+func testBuildDetailsLink(t *testing.T, sb integration.Sandbox) {
+	buildDetailsPattern := regexp.MustCompile(`(?m)^View build details: docker-desktop://dashboard/build/[^/]+/[^/]+/[^/]+\n$`)
+
+	// create desktop-build .lastaccess file
+	home, err := os.UserHomeDir() // TODO: sandbox should create a temp home dir and expose it through its interface
```

Comment:
I think this should still be "safe" to merge without this TODO?

No other tests access this file, so should be alright for now? Though obviously good to follow up asap.

Reply:
Yes fine for now 👍